package.json

```diff
@@ -18,8 +19,7 @@
     "start": "HTTPS=true react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "precommit": "NODE_ENV=production lint-staged"
```

Comment:
Should be left here and I really commend you to add more eslint rules or using `/* eslint disable ... */`.